src/app/src/main/kotlin/org/imperial/mrc/hint/security/SecureDbProfileService.kt

```diff
@@ -0,0 +1,20 @@
+package org.imperial.mrc.hint.security
+
+import org.pac4j.core.credentials.password.JBCryptPasswordEncoder
+import org.pac4j.core.credentials.password.PasswordEncoder
+
+class SecurePasswordEncoder : PasswordEncoder {
```

Comment:
Could add a test for these methods together I guess - test that matches(x, encode(x)) is true